### PR TITLE
Add documentation of MDNS

### PR DESCRIPTION
**Pull request by husigeza**
MDNS feature is released with PR: https://github.com/pycom/pycom-micropython-sigfox/pull/377
Adding documentation for it.

### CODE REVIEW

File: content/firmwareapi/pycom/network/mdns.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+* `hostname` is the hostname to set
+* `instance_name` is the instance name to set
+
+#### MDNS.add_service(service_type, proto, port, \*, txt)
+
+Adds a service to the MDNS module which will be advertised.
+
+The arguments are:
+
+* `service_type` is the type of the offered service, e.g.: _http, _ftp or can be custom service
```
Comment (lilycey) [inline content/firmwareapi/pycom/network/mdns.md:87]: The formatting here seems slightly different - will it upload well?

**Review by lilycey — APPROVED** [review]
All good bar one small question!

File: content/firmwareapi/pycom/network/mdns.md (hunk 1 above)
Comment (husigeza) [reply content/firmwareapi/pycom/network/mdns.md:87]: I checked and the content of the mdns.md is exactly the same as what we have on the publish-development branch. Is that a problem?

**Comment by jirkadev** [comment]
If this is the stable firmware feature, we should open the second PR and merge this branch also to development-publish

I wrote a script which can open two PRs automatically and assign a reviewer to it here
https://github.com/pycom/pycom-documentation/tree/publish/scripts

**Comment by husigeza** [comment]
The documentation of this feature has already been merged into development-publish: https://github.com/pycom/pycom-documentation/pull/183

**Pull request merged**